client/src/components/existing-request/existing-request-display.vue

```diff
@@ -667,6 +667,19 @@ export default class ExistingRequestDisplay extends Mixins(
     }
   }
 
+  private cancelledUpgrade (status: string, payments: any): string {
+    let paymentId = null
+    if (status === 'PAYMENT_CANCELLED') {
```

Comment:
Are you sure the status is this string exactly? It comes from the API right?

It would be nice to use a type/enum for this, so that we have a single place where the API codes (or URL query parameters, in this case) are declared. On the other hand, if this is the only place this code is used then that's overkill...

Reply:
Yeah I was thinking that too haha. It is that string exactly and we don't use it anywhere else

Reply:
OK, it _should_ be an enum to be explicit that this is a code we receive from an API (pay)...

But don't change this now unless you're already updating that code.